docs/comb-msi/mux-functions.md

```diff
@@ -62,3 +62,16 @@ The set $\\{M(1),0,1\\}$ is functionally complete (See section **Functionally Co
 ## References
 
 {% bibliography --cited --file books --file mux %}
+
+{:.quiz}
+
+  1. Can one MUX block be said an universal logic block ?
+     1. Yes
+     * No
+  2. Can one DEMUX block be said an universal logic block ?
+     1. Yes
+     * No
+  3. Which of the following option is correct regarding MUX-based gates ?
```

Comment:
Question can be rephrased as something like "Select the MUX-based gates which is/are possible"